### ISSUE TITLE
Decide on how to localize content on mozilla.org in the future.

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#6774]

**Comment by hoosteeno** [comment]
The draft PRD is as far as we can get it without some technical input. https://docs.google.com/document/d/1uP7ke5KKrI8dkm2fzIn4TMvdC51jtcWL8Dgt-rm5QGI/edit#

@pmac, any update on Fluent, Gettext, and the architecture of localization going forward?

**Comment by pmac** [comment]
Yes. My apologies for the lack of movement. SUMO has taken more time than expected.

I think Fluent is the right way to go for us. It will mean a significant workflow change for the team, but I believe it will come with enough benefit to go with it. It's seen a 1.0 release recently, and has good momentum and adoption throughout the company. I would like to discuss this with the dev team before we make that final decision, but Fluent is my current vote.

**Comment by alexgibson** [comment]
Draft of system requirements and proposed workflows: https://docs.google.com/document/d/1zArroCmtY_ql4Eo56GvibYbB3DLr1aWb5g1_J4Nb7GQ/edit?usp=sharing
